netwaystev2/src/transport/transport.rs

```diff
@@ -21,14 +21,14 @@ use tokio::time as TokioTime;
 use tokio_stream::wrappers::IntervalStream;
 use tokio_util::udp::UdpFramed;
 
-type TransportCmdSend = Sender<TransportCmd>;
+pub type TransportCmdSend = Sender<TransportCmd>;
 type TransportCmdRecv = Receiver<TransportCmd>;
 
 type TransportRspSend = Sender<TransportRsp>;
-type TransportRspRecv = Receiver<TransportRsp>;
+pub type TransportRspRecv = Receiver<TransportRsp>;
 
 type TransportNotifySend = Sender<TransportNotice>;
```

Comment:
Should these be `pub` as well?

Reply:
Possibly. Right now it's only those that cross the Filter/Transport bridge.